tools/__tasks__/compile/javascript/index.js

```diff
@@ -1,12 +1,12 @@
-module.exports = [{
-    title: 'Compile JS',
+module.exports = {
+    description: 'Compile JS',
     task: [
         require('./clean'),
-        require('../inline-svgs/compile'),
+        'compile/inline-svgs',
```

Comment:
I am unclear on when you would use `'compile/inline-svgs'` instead of `require('./compile/inline-svgs')`. Are the two essentially interchangeable?